### PR TITLE
Specific azure requirements versions.

### PR DESCRIPTION
**Pull request by smarlowucf**
Fix build error due to azure version requirements.

### CODE REVIEW

File: .virtualenv.requirements.txt [hunk 1]
```diff
@@ -5,6 +5,8 @@
 # $ pip install -U pip setuptools
 # $ pip install -r .virtualenv.requirements.txt
 
+azure-mgmt-compute~=3.0
+azure-mgmt-resource~=1.2
```
Comment (schaefi) [inline .virtualenv.requirements.txt]: We also need the storage components

* azure-storage

**Comment by schaefi** [comment]
I also realized the spec file ```package/mash-spec-template``` is missing appropriate requires for the packages. Could you also add:

* python3-azure-sdk

Or do we want to require the component packages:

* python3-azure-mgmt-resource
* python3-azure-mgmt-compute
* python3-azure-storage

Thanks

**Comment by smarlowucf** [comment]
> Or do we want to require the component packages:

I prefer requiring only the components needed.

With azure-storage are we okay with slightly different requirements with spec and pypi or should they match? Azure has storage split up but we don't yet. The component requirement is azure-storage-blob which is in pypi but we only package the meta azure-storage.

**Comment by schaefi** [comment]
>With azure-storage are we okay with slightly different requirements with spec and pypi or should they >match?

yes I think that's ok

**Comment by smarlowucf** [comment]
> yes I think that's ok

Good to merge and fix build?

**Pull request merged**